### PR TITLE
Mobile second section

### PR DESCRIPTION
**Pull request by AnelisaDee**
**This is a portfolio mobile version.**
This project is developed using best practice of linters. I created the sections of this mobile website using the template found in Figma App.

**Project Requirements**

 **Build the middle section of the mobile website**

- I used css grids and flexbox container for the middle section in order for it to be more responsive in any device.

- Styled it according to the template I chose.

- The 'work' and 'About Me' section

- The content of the body is container in one grid container

- Elements inside the grid are in flexbox

- Added 2 space indentation inside the .css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,5 +48,86 @@ <h2>I’am a Software Developer</h2>
...
+      <div class="work-list">
+        <ul>
+          <a href="#"><li class="item work-item-1">css</li></a>
+          <a href="#"><li class="item work-item-2">html</li></a>
+          <a href="#"><li class="item work-item-3">bootstrap</li></a>
+          <a href="#"><li class="item work-item-4">Ruby</li></a>
+        </ul>
+      </div>
+      <div class="work-link">
+        <a href="#">See Project</a>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `required`
- [ ] Good job so far 👏 
- [ ] Please make you increase some space on `most recent works` also and an underline like the Figma template
- [ ] Also consider making sure that the `see project` button, the words don't fold like the Figma template
- [ ] You may also consider giving the card the right background color the Figma template
- [ ] please screenshots below


`your template`
image


`figma template`
image

File: index.html [hunk 2]
```diff
@@ -47,5 +48,86 @@ <h2>I’am a Software Developer</h2>
...
+        <ul>
+          <a href="#"><li class="project-item-1">html</li></a>
+          <a href="#"><li class="project-item-2">bootstrap</li></a>
+          <a href="#"><li class="project-item-3">Ruby</li></a>
+        </ul>
+      </div>
+      <div class="project-list">
+        <a href="#">See Project</a>
+      </div>
+      </div>
```
Comment (mwanawabangona) [inline index.html:173]: - [ ] `required`
- [ ] please consider a space between the 2 cards the Figma template
- [ ] also you may consider making the button stretch all the way as the Figma template
- [ ] please see screenshots belows

`your template`
image

`figma tempate`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @AnelisaDee,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

- [ ] `required`
- [ ] Please add more cards you have only made a few please add just as the figma template has 7 cards

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -47,5 +48,88 @@ <h2>I’am a Software Developer</h2>
...
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      </div>
+      <div class="project-lists">
+        <ul>
+          <a href="#"><li class="project-item-1">html</li></a>
+          <a href="#"><li class="project-item-2">bootstrap</li></a>
+          <a href="#"><li class="project-item-3">Ruby</li></a>
+        </ul>
+      </div>
```
Comment (Gopxfs) [inline index.html:169]: As requested by the previous reviewer:
Please update your project cards to be equal to the ones on Figma. Kindly notice that Figma template has 6 cards:
image

While your project only has one:
image
> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

File: style.css [hunk 4]
```diff
@@ -1,136 +1,690 @@
...
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  padding: 12px 12px;
+}
```
Comment (Gopxfs) [inline style.css]: As requested by the previous reviewer, please make sure that your button text is displayed only in one line, as in the Figma design 😃:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 5]
```diff
@@ -47,5 +48,88 @@ <h2>I’am a Software Developer</h2>
         </div>
     </section>
     </div>
+    <section class="grid-work">
+      <div class="work-sec">
+      <div class="work-heading"><h1>My Recent Works</h1>
+      <hr>
+      </div>
```
Comment (Gopxfs) [inline index.html:55]: Great job adding the line as requested! To fit with the original design, please make sure that it looks like the one in the template:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

File: style.css [hunk 6]
```diff
@@ -1,136 +1,690 @@
...
+.project-list {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: #ff6b00;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+}
```
Comment (Gopxfs) [inline style.css]: Amazing job adjusting the margin and the button width 😄! Please remove the underline on the button text to make it fit the one in Figma 😉.
image

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @AnelisaDee 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Good folder structure.
✔️ Added some changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUEST CHANGES :recycle: 
## Hi @AnelisaDee    :hand: :hand: 
I am @tuyishimireeric your reviewer for this PR :man_beard:

Great work so far! 👏
Thank you for implementing most of the required changes by the previous reviewer 👏  however, there are some issues that you still need to work on to go to the next project, but you are almost there!
you-are-near


## To highlight 🎯

:heavy_check_mark:  Gitflow used. :+1: 
:heavy_check_mark: No Linter errors :+1: 
:heavy_check_mark: Descriptive commit messages :+1: 

## Required Changes ♻️

- [ ] Please you still have a small issue on your button, it does look like that of figma design
image
Simply remove ```letter-spacing: 0.03em;``` in your css codes :joy: only this will make your button like that of Figma design. :+1:  
image


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (TuyishimireEric) [reply style.css]: - [ ] Good job so far! :clap: but as requested by the previous code reviewer,  your button text should look like that of Figma design :smile:  you only need to remove ```letter-spacing: 0.03em;``` to put your text on one line in your button :+1:

**Review by OyePriscilla — APPROVED** [review]
# Hi @AnelisaDee  😄 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 



## Highlights
The commit messages are meaningful ✅
Linters are working properly ✅
Github flow observed ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**